lib/socrates/core/dispatcher.rb

```diff
@@ -151,8 +152,10 @@ def done_transitioning?(state)
       end
 
       def handle_action_error(error, session, state)
-        @logger.warn "Error while processing action #{state.data.state_id}/#{state.data.state_action}: #{error.message}"
-        @logger.warn error
+        @error_logger.error "Error while processing action \
+          #{state.data.state_id}/#{state.data.state_action}: #{error.message}"
+        @error_logger.error "For user: #{session.user}"
+        @error_logger.error error
```

Comment:
To Christian's point, we should definitely roll these up into one error regardless of how long the error message is, so there is only one error showing up in Rollbar.